collection/Griffin Macaulay; The Griffon's Saddlebag, Book 2.json

```diff
@@ -22107,6 +22125,200 @@
 			]
 		}
 	],
+	"subclassFluff": [
+		{
+			"name": "Path of the Infernal",
```

Comment:
None of these will be picked up without accompanying `"hasFluffImages": true` on each `subclass`

Reply:
Picked up by what? Plutonium? Looks fine in 5etools. (Regardless, I think I am going to move everything from subclassfluff as fluff under subclass).
<img width="869" alt="image" src="https://github.com/user-attachments/assets/3c0a8f3c-b40b-4694-b800-11c9ac4c496f">